test/valid-json-arrays-in-bash.js

```diff
@@ -0,0 +1,74 @@
+const { expect } = require("chai");
+const validJsonArrayInBashCheck = require("../checks/valid-json-arrays-in-bash");
+
+["CMD", "ENTRYPOINT"].map(varName => {
```

Comment:
Smart. I've never thought to use `describe` in a `map` like this

Reply:
Thanks!